Prepare release for version `2.0.0-beta.6` (#1224)
microsoft/mcp: servers/Azure.Mcp.Server/vscode/CHANGELOG.md

# Release History

> ## 2.0.6 (2025-11-19) (pre-release)

Author: This is following the pattern of versioning seen in this particular changelog file, copilot should be analyzing patterns seen in same file not in different files

# Release History

## 2.0.6 (2025-11-19) (pre-release)

### Added

> - Added a [hidden] command `server_info` to provide server information (name, version) so server metadata is programmatically parsed in telemetry. [[#1164](https://github.com/microsoft/mcp/pull/1164/)]

Reviewer: ```suggestion
- Added a [hidden] command `server_info` to provide server information (name, version) so server metadata is programmatically parsed in telemetry. [[#1164](https://github.com/microsoft/mcp/pull/1164)]
```

> - Enabled HTTP trace telemetry for self-hosted Azure MCP server. Added OpenTelemetry tracing support with Azure Monitor exporter for HTTP transport mode, allowing self-hosted instances to export traces to Application Insights when `APPLICATIONINSIGHTS_CONNECTION_STRING` is configured. [[#1227](https://github.com/microsoft/mcp/pull/1227)]

Reviewer: ```suggestion
- Added a [hidden] command `server_info` to provide server information (name, version) so server metadata is programmatically parsed in telemetry. [[#1164](https://github.com/microsoft/mcp/pull/1164/)]
- Added OpenTelemetry tracing support with Azure Monitor exporter for HTTP transport mode, allowing self-hosted instances to export traces to Application Insights when `APPLICATIONINSIGHTS_CONNECTION_STRING` is configured. [[#1227](https://github.com/microsoft/mcp/pull/1227)]
```

### Changed

- We now capture information for the MCP client request's `_meta` store. [[#1154](https://github.com/microsoft/mcp/pull/1154)]

> - Renamed Microsoft Azure AI Foundry to Microsoft Foundry. [[#1211](https://github.com/microsoft/mcp/pull/1211/)]

Reviewer: ```suggestion
- Renamed Microsoft Azure AI Foundry to Microsoft Foundry. [[#1211](https://github.com/microsoft/mcp/pull/1211)]
```
